samples/emoji-search/presenters/build.gradle.kts

```diff
@@ -4,11 +4,14 @@ apply(plugin = "app.cash.zipline")
 
 plugins {
   kotlin("multiplatform")
+  id("com.android.library")
   kotlin("plugin.serialization")
 }
 
 kotlin {
-  jvm()
+  android {
```

Comment:
switching from JVM to Android as we now need an Android context to create a ZiplineLoader on any Android platform. This is because we use Android’s sqlite in the cache!